### DISCUSSION TITLE
Load Full Values taking hour+

### DISCUSSION THREAD

**Discussion by suehshtri** [pawelsalawa/sqlitestudio#4293] [Q&A]
I have some string results that appear in the UI truncated.  Less than a 1000 rows.  When I click the "Load Full Values" , SqliteStudio 3.3.3 locks up, sometimes for more than an hour, before it comes back with the rows.  Once it does it seems fine.
Is there something I can do to get around this?
Is there a setting for the truncation length for the UI?

**Comment by pawelsalawa** [comment]
I assume this is a SQL query results (not a Table Window), right? Would you be able to share the query?

**Reply by suehshtri** [reply]
I would have to contrive something I can share.

**Reply by PatriaMinas** [reply]
Are you running a sql query? Show us just a screenshot of the problem. You don't need to show us your data, but you can show us the select (sql query) and a screenshot of the problem. Or are you referring to some other feature of the software?
Thank you.

**Reply by suehshtri** [reply]
I am viewing SQL query results, several joins, some aggregation, and a text column that can be fairly wide.  I will try to get work up a repro for you.  I had to fall back to a custom client.

**Reply by pawelsalawa** [reply]
It's clear to me now that the issue is the same as discussed in #4129 - which is fixed for upcoming 3.4.0 release.
